net/http/impl/server.h

```diff
@@ -456,7 +457,7 @@ class HTTPServerConnection final {
           connection_.BlockingWrite("0", true);
           connection_.BlockingWrite(kCRLF, false);
           connection_.BlockingWrite(kCRLF, false);  // We should send CRLF twice.
-        } catch (const std::exception& e) {  // LCOV_EXCL_LINE
+        } catch (const std::exception& e) {         // LCOV_EXCL_LINE
```

Comment:
That's what `clang-format` does. I used to fix it back for a few times, decided to not fight the inevitable this time :-)